packages/webclient/src/AuthHandler/AzureEndpoint.ts

```diff
@@ -90,12 +92,12 @@ export class AzureEndpoint implements IAuthorizationHandler {
             }
         };
 
-        let webRequestOptions: webClient.WebRequestOptions = {
-            retriableStatusCodes: [400, 408, 409, 500, 502, 503, 504]
+        let webRequestOptions: WebRequestOptions = {
+            retriableStatusCodes: [408, 409, 500, 502, 503, 504]

```

Comment:
why have we removed 400?

Reply:
400 is not a retriable error code. Ideally only 5xx errors should be retried.

Reply:
can you check why was this added in tasks. If not required there, raise a bug to clean it up.

Reply:
Please add your findings here

Reply:
not sure as to why this was added in tasks. 400 represents a malformed request and retrying will only waste time. I will open up a story to clean it up in task.